modules/docs/arrow-docs/docs/docs/datatypes/statet/README.md

```diff
@@ -136,7 +136,7 @@ import arrow.instances.either.monadError.*
 
 fun stackOperationsS2() = 
  ForStateT<EitherPartialOf<StackError>, Stack, StackError>(Either.monadError<StackError>()) extensions {
-  binding {
+  binding<StateTPartialOf<EitherPartialOf<StackError>, Stack>, String> {
```

Comment:
The compiler trips here if the types are redundantly not specified. This is unrelated to Ank.